news/c8285e11-5b65-498c-9eaa-85ca096243f9.trivial.rst

```diff
@@ -0,0 +1 @@
+Upgrade distro to 1.6.0.
```

Comment:
This needs to use `.vendor.rst` (IIRC; please double check the config in `pyproject.toml`)

Reply:
Thanks. Updated in the latest revision.